scripts/run-goto-transcoder.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+
+##############
+# PARAMETERS #
+##############
+contract_folder=target/kani_verify_std/target/x86_64-unknown-linux-gnu/debug/deps
```

Comment:
nit... can you pass this as an argument to Kani script and the goto-transcoder script instead of relying here on the location where Kani stores their temporary files?

Reply:
You can pass `--target-dir` to Kani to select where to put the artifacts.

Reply:
I added the flag to the script